### PR TITLE
Webui: extend led hardware config + connection lost page

### PR DESCRIPTION
**Pull request by redPanther**
**1.** Tell us something about your changes.
as the title said ...
+ automatic reconnect after connection lost.
+ connection lost is triggered after 3 times connection error to avoid  false detection
+ implement led colors streaming with 4 fps

**2.** If this changes affect the .conf file. Please provide the changed section

**3.** Reference an issue (optional)

Note: For further discussions use our forum: forum.hyperion-project.org

### CODE REVIEW

**Comment by brindosch** [comment]
amazing!

note: modal for show led config check result is no longer working (or the check itself)

**Comment by redPanther** [comment]
> note: modal for show led config check result is no longer working (or the check itself)
I will remove button and make the check when saving ...

File: assets/webconfig/css/hyperion.css [hunk 1]
```diff
@@ -13,7 +13,7 @@ body{font-family:Roboto,"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:15
 table.borderless td,table.borderless th{border: none !important;}
 
 /*Header*/
-.navbar-brand{padding: 5px;height:90px;}
+.navbar-brand{padding: 5px;padding-left:20px;height:60px;}
```
Comment (brindosch) [inline assets/webconfig/css/hyperion.css]: 60px, sure? Now the sidebar should overlap with the navbar? No?

**Comment by brindosch** [comment]
I forgot to add V4L to the components list at the components page. You could add it if you like (also to prevent loss of this change :)

**Comment by redPanther** [comment]
now led colors view is nearly realtime ... we stream ledcolors in 250ms (=4fps) interval to webui without significant performance increase.

**Comment by brindosch** [comment]
you removed the line numbers from the editor?

**Comment by redPanther** [comment]
I disbaled the editor plugin, it has to many side effects .... we need a better one.

**Comment by redPanther** [comment]
.... and now with compontns fully integrated (but we need some refactoring of component stuff. current impl. is okay, but not good enough for future)

**Comment by Paulchen-Panther** [comment]
thanks redPanther

**Pull request merged**